### PR TITLE
chore: Fix error console logs observed

### PR DESCRIPTION
**Pull request by escobarjonatan**

### CODE REVIEW

File: packages/elements/src/lib/TextInput.tsx [hunk 1]
```diff
@@ -165,7 +165,10 @@ export const TextInput: ForwardRefRenderFunction<
           fontWeight={ 500 }
         >
           <>
-            <Typography sx={ { fontWeight: 500, textTransform: "uppercase" } }>
+            <Typography
+              component="span"
+              sx={ { fontWeight: 500, textTransform: "uppercase" } }
+            >
```
Comment (escobarjonatan) [inline packages/elements/src/lib/TextInput.tsx:171]: We were using a typography inside another typography so a `p` inside a `p`:
image

File: packages/elements/src/lib/form/CopyrightInputField.tsx [hunk 2]
```diff
@@ -58,7 +59,7 @@ const CopyrightInputField: ForwardRefRenderFunction<HTMLDivElement, Props> = (
           } }
         >
           <Stack direction="row" spacing="4px">
-            <Box>{ label }</Box>
+            <Box component="label">{ label }</Box>
```
Comment (escobarjonatan) [inline packages/elements/src/lib/form/CopyrightInputField.tsx:62]: Placing a div inside a `p` = no bueno

image

File: packages/elements/src/lib/form/FormProgressStepper.tsx [hunk 3]
```diff
@@ -78,8 +78,8 @@ const FormProgressStepper = ({
 
         return (
           <StepBox
-            boxColor={ stepColor }
-            boxTitle={ stepTitle }
+            boxcolor={ stepColor }
+            boxtitle={ stepTitle }
```
Comment (escobarjonatan) [inline packages/elements/src/lib/form/FormProgressStepper.tsx:82]: case-sensitive 

image

File: apps/studio/src/pages/home/library/SongList.tsx [hunk 4]
```diff
@@ -279,10 +279,10 @@ export default function SongList({ totalCountOfSongs, query }: SongListProps) {
       return song.id === playerState.currentPlayingSongId;
     });
 
-    if (!isSongFound) {
+    if (!isSongFound && !isLoading) {
       stopSong();
     }
-  }, [playerState.currentPlayingSongId, songData, stopSong]);
+  }, [playerState.currentPlayingSongId, songData, stopSong, isLoading]);
```
Comment (escobarjonatan) [inline apps/studio/src/pages/home/library/SongList.tsx:285]: logic was running while loading, added a check to prevent it:
image

**Review by dmkirshon — APPROVED** [review]
LGTM

**Review by scandycuz — APPROVED** [review]

**Pull request merged**